app/src/main/res/xml/preferences.xml

```diff
@@ -62,20 +62,12 @@
         <Preference
             android:key="FAQ"
             android:summary="@string/FrequentlyAskedQuestions"
-                    android:title="@string/FAQ"/>
-
-            
-           <Preference android:title="@string/translate_help_title"
-        android:key="@string/lang_translate"
-        android:summary="@string/translate_help_summary"/>
-    <Preference
-
-        android:title="@string/Terms"
-        android:summary="@string/TermsOfUse"
-        android:key="@string/Terms"/>
-
-          
+            android:title="@string/FAQ" />
 
+        <Preference
+            android:key="@string/Terms"
```

Comment:
Please be careful that this is not a translated string

Reply:
I didn't change that part, left it as it is. However, I can change it to a Hardcoded string although it is not related to this issue and send a PR again if you like

Reply:
Oh I am sorry, my mistake :)

Reply:
Yeah sure I will change it to "Terms" and send a PR again :)